src/stats.jl

```diff
@@ -435,10 +432,10 @@ function summarystats(chn::Chains;
 
     # Store everything.
     funs = [mean∘cskip, std∘cskip, sem∘cskip, df_mcse]
-    func_names = [:mean, :std, :naive_se, :mcse]
+    func_names = [:mean, :std, :naivese, :mcse]
```

Comment:
Shouldn't this have an underscore since it's two words? It's a little hard to read.

Reply:
Yes, I agree, I think in this case the underscore actually improves readability. Alternatively, one could also write a bit more explicitly `:naivestandarderror` (with or without underscore?), `:standarddeviation` (or `:stddev`?), and `:mcstandarderror` or `:montecarlostandarderror`. Mhh but maybe we should keep it shorter :smile:

Reply:
Yeah, I think `naive_se` is short and readable. I'm fine with `stddev` or `std`, I'm very familiar with both and I think nearly all R users would agree with `std`.